rg_instructor_analytics/management/commands/__init__.py

```diff
@@ -0,0 +1 @@
+# flake8: noqa
```

Comment:
Why do we have this line in the init file?

Reply:
For avoiding error from flake8 about missing docstring for the public module.